src/Symfony/Component/HttpKernel/composer.json

```diff
@@ -20,7 +20,7 @@
         "symfony/deprecation-contracts": "^2.1|^3",
         "symfony/error-handler": "^4.4|^5.0|^6.0",
         "symfony/event-dispatcher": "^5.0|^6.0",
-        "symfony/http-foundation": "^5.3.7|^6.0",
+        "symfony/http-foundation": "^5.4|^6.0|^6.1|^6.2",
```

Comment:
You must specify the patch versions here. For 5.4 for instance, that would be 5.4.21 as this is going to be the next version.

Reply:
Where can I find which patch versions are needed? I'm not really familiar with this part so I'm a bit lost what the correct solution should be here. Or is it just the next patch version for all release branches?

Reply:
You can have a look at the tags and add +1. I've done it for you. So, everything is ok now.
Note that I've removed 6.0 and 6.1 here as they are not maintained anymore and as such will never have this patch.

Reply:
Thank you